Session 2 - HeteroRecyclerView.md (#53)
Cyber-Labs/Session-Reports: Android Development/2019-20/Session 2 - HeteroRecyclerView.md

# Session 2

> Hetero RecyclerView

Reviewer: ```suggestion
Heterogeneous RecyclerView
```

Reviewer: do it for others as well

Reviewer: still pending

> Conducted on: 24/02//2020

Reviewer: ```suggestion
Conducted on: 24/02/2020
```

## Agenda
Learning the various features, uses and implementation of Hetero RecyclerView in Android Applications.

## Summary

> Discussion was held on Hetero RecyclerView, and how to use it, and dessign it was taught. Frequent mistakes in its implementation and its flow of execution was also discussed. Towards the end, discussion was held on doing projects and activeness.

Reviewer: ```suggestion
The discussion was held on how to use and design Heterogeneous RecyclerView. Frequent mistakes in its implementation and its flow of execution was also discussed. Towards the end, the discussion was held on doing projects and activeness.
```
